scripts/dqlite/Makefile

```diff
@@ -134,8 +134,11 @@ musl-install:
 musl-cross-arch-install:
 	@PATH=${PATH}:${MUSL_CROSS_BIN_PATH} which musl-gcc >/dev/null && { echo "musl already installed ${MUSL_CROSS_PATH}"; exit 0; }
 
-	@mkdir -p ${MUSL_CROSS_PATH}
-	git clone git@github.com:richfelker/musl-cross-make.git ${MUSL_CROSS_PATH}
+	echo "MUSL_CROSS_PATH=${MUSL_CROSS_PATH}"
+	echo "PATH $(shell pwd)"
```

Comment:
This is useful for debugging, but eventually can be removed once we know the failure patterns.